src/Scene/NodeProcess.js

```diff
@@ -479,16 +479,19 @@ NodeProcess.prototype.processNode = function processNode(node, camera, params) {
  * @returns {NodeProcess_L7.NodeProcess.prototype.frustumCullingOBB.node@pro;camera@call;getFrustum@call;intersectsBox}
  */
 
-var quaternion = new THREE.Quaternion();
+const frustum = new THREE.Frustum();
+const obbMatrixWorld = new THREE.Matrix4();
 
 NodeProcess.prototype.frustumCullingOBB = function frustumCullingOBB(node, camera) {
-    // position in local space
-    var position = node.OBB().worldToLocal(camera.position().clone());
-    position.z -= node.distance;
+    // Move camera in OBB local space
+    obbMatrixWorld.copy(camera.camera3D.matrixWorldInverse);
```

Comment:
[Three's scene documentation](https://threejs.org/docs/index.html?q=scene#Reference/Scenes/Scene) seems to indicate that autoUpdate has priority.

It would be a good idea to remove all the `matrixAutoUpdate = false` since they are useless.